src/lib/import-export/import/importers/importer.ts

```diff
@@ -162,12 +168,40 @@ export class JetpackImporter extends BaseImporter {
 	}
 }
 
+export class LocalImporter extends BaseBackupImporter {
+	protected async parseMetaFile(): Promise< MetaFileData | undefined > {
+		const metaFilePath = this.backup.metaFile;
+		if ( ! metaFilePath ) {
+			return;
+		}
+		this.emit( ImportEvents.IMPORT_META_START );
+		try {
+			const metaContent = await fsPromises.readFile( metaFilePath, 'utf-8' );
+			const meta = JSON.parse( metaContent );
+			const phpVersion = meta?.services?.php?.version;
+			return {
+				phpVersion: phpVersion.split( '.' ).slice( 0, 2 ).join( '.' ) ?? '8.1',
```

Comment:
Local keeps the version in `x.y.z` format, but Studio accepts only `x.y`.  Is there a better way to handle it than parsing string?

Reply:
Thanks, I applied this one: https://github.com/Automattic/studio/pull/436/commits/e77de67bfe07792b530f9b431006d5c8bc14b455